src/POData/UriProcessor/ResourcePathProcessor/SegmentParser/TargetKind.php

```diff
@@ -23,6 +23,10 @@
  */
 class TargetKind extends Enum
 {
+    protected const TERMINAL_VALUES = [6 => true, 7 => true, 9 => true, 11 => true, 12 => true];
+    protected const DIRECT_PROCESS_VALUES = [2 => true, 7 => true, 9 => true];
+    protected const FILTERABLE_VALUES = [3 => true, 4 => true];
```

Comment:
I think this is round about. those values are singletons and as such NOT Filterable value